mediaire_toolbox/transaction_db/model.py

```diff
@@ -207,7 +207,7 @@ def read_dict(self, d: dict):
         self.data_uploaded = self._str_to_datetime(d.get('data_uploaded'))
         self.billable = d.get('billable')
         self.priority = d.get('priority')
-        self.site_id = d.get('site_id')
+        self.site_id = d.get('site_id', 0)
```

Comment:
If we don't set a defautl here, will the `site_id = Column(Integer, default=0)` in the class definition still fill this in? I don't like having defaults in multiple places, but it is really important to not mess up the site_id

Reply:
No, this default will only be applied when reading messages that do not contain the field (older messages)